src/app/members/components/MemberList.tsx

```diff
@@ -24,16 +24,21 @@ import { Member } from "@/types";
 import { UserStatusBadge } from "@/common/StatusBadge/UserStatusBadge";
 import { StandardPage } from "@/common/StandardPage/StandardPage";
 import { Routes } from "@/lib/routes";
+import { formattedPhoneNumber } from "@/utils/formattedPhoneNumber";
+import { formattedRole } from "@/utils/memberRoleFormatter";
 
 interface Props {
   data: Member[];
   success: boolean;
   message: string;
 }
+
 export const MemberList = ({ members }: { members: Props }) => {
   const { toast } = useToast();
-  const { data, success } = members;
+  const { data : membersList, success } = members;
```

Comment:
I'm not a fan of any variable name with list. Also, why are you getting `success` and `message` from props? I thought we removed that a long time ago. @nadirhussain786

Reply:
Sir I have fixed , @nadirhussain786 i have assigned you a ticket for this [TICKET](https://github.com/Monfernape/shoes-rack/issues/217#issue-2698376510)